package.json

```diff
@@ -3,18 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.0.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "autoprefixer": "^9.8.6",
+    "postcss": "^7.0.35",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.4"
+    "react-scripts": "^4.0.1",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.2"
```

Comment:
What are the cost-benefits for using tailwindcss considering the size of the project and change?

Reply:
There are a lot of benefits
1. It has a built-in design system
2. We won't be mixing inline styling with CSS stylesheet.
3. It is easier to implement responsive styling with Tailwind, not need to be worrying about media query.
4. Css caching